server/command_hooks.go

```diff
@@ -405,6 +417,38 @@ func getDialogWithIntroductionText(introductionText string) model.Dialog {
 	return dialog
 }
 
+func (p *Plugin) executeCommandInteractive(args *model.CommandArgs) *model.CommandResponse {
+
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.botId,
```

Comment:
Please add to fix the second issue
```suggestion
		RootId:    args.RootId,
		UserId:    p.botId,
```